packages/e2e-tests/src/specs/dashboard/publisherLogo.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  uploadPublisherLogo,
+  visitSettings,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('publisher logo', () => {
+  afterEach(async () => {
+    // Deletes all publisher logos using the browser context
+    await page.evaluateHandle(async () => {
+      await wp.apiFetch({
+        path: '/web-stories/v1/settings/',
+        method: 'POST',
+        data: {
+          web_stories_publisher_logos: [],
+          web_stories_active_publisher_logo: -1,
+        },
+      });
+    });
```

Comment:
Instead of doing this, can we delete all publisher logos in the e2e or use a e2e plugin to delete them in PHP.

Reply:
Sure, there's no way to do it in the app (prevents you from deleting the last one) so we needed to do it a different way. Looks like you posted a util called `deleteMedia`, so I'll take a look into that